src/System.Windows.Forms/src/System/Windows/Forms/TextBoxBase.TextBoxBaseUiaTextProvider.cs

```diff
@@ -205,7 +205,14 @@ public override LOGFONTW Logfont
 
         public override string Text
             => Owner is not null && Owner.IsHandleCreated
-                ? Owner.Text
+                ? Owner.PasswordProtect ? PasswordString
+                : Owner.Text
+                : string.Empty;
```

Comment:
Consider indenting each branch
```suggestion
                ? Owner.PasswordProtect 
                    ? PasswordString
                    : Owner.Text
                : string.Empty;
```

Reply:
Done.